src/main/java/com/opinta/mapper/CounterpartyMapper.java

```diff
@@ -4,8 +4,9 @@
 import com.opinta.entity.Counterparty;
 import org.mapstruct.Mapper;
 import org.mapstruct.Mapping;
+import org.springframework.stereotype.Component;
 
-
+@Component
```

Comment:
There was annoying error from ide, that said that it can't inject bean. Adding these @Component annotations fixed the problem.